client/PublicRequestTypes.ts

```diff
@@ -652,7 +660,7 @@ export type ResultByRequestType<T> =
     T extends RequestType.ONBOARD | RequestType.SIGNUP | RequestType.LOGIN
             | RequestType.MIGRATE | RequestType.LIST ? Account[] :
     T extends RequestType.LIST_CASHLINKS ? Cashlink[] :
-    T extends RequestType.CHOOSE_ADDRESS | RequestType.ADD_ADDRESS ? Address :
+    T extends RequestType.CHOOSE_ADDRESS | RequestType.ADD_ADDRESS ? ChooseAddressResult :
```

Comment:
`RequestType.ADD_ADDRESS` should still just return an `Address`.

Reply:
oh, you are right! Overlooked this.